opencga-core/src/main/java/org/opencb/opencga/core/common/MailUtils.java

```diff
@@ -33,19 +35,18 @@ public class MailUtils {
     private static final Logger logger = LoggerFactory.getLogger(MailUtils.class);
 
     public static void sendResetPasswordMail(String to, String newPassword, final String mailUser, final String mailPassword,
-                                             String mailHost, String mailPort) throws Exception {
-        sendResetPasswordMail(to, newPassword, mailUser, mailPassword,
-                mailHost, mailPort, "true");
-    }
-
-    public static void sendResetPasswordMail(String to, String newPassword, final String mailUser, final String mailPassword,
-                                             String mailHost, String mailPort, String ssl) throws Exception {
+                                             String mailHost, String mailPort, String ssl, String user_id) throws Exception {
 
         Properties props = new Properties();
         props.put("mail.smtp.auth", "true");
-        props.put("mail.smtp.starttls.enable", ssl);
+        props.put("mail.smtp.ssl.enable", ssl);
         props.put("mail.smtp.host", mailHost);
         props.put("mail.smtp.port", mailPort);
+        props.put("mail.smtp.starttls.enable", "true");
+        props.put("mail.smtp.starttls.required", "true");
+        props.put("mail.smtp.ssl.protocols", "TLSv1.2");
+        props.put("mail.smtp.ssl.checkserveridentity", "true");
+        props.put("mail.smtp.socketFactory.class", "javax.net.ssl.SSLSocketFactory");
```

Comment:
All the variables that are set in the code and are not in the configuration are necessary for the current version to work..... The objective of this task was to change the body of the email message. In the tests I discovered that it did not work with the current configuration, I have added those properties to make it work, if you want to bring that to the configuration it must be done in another requirement, which must be planned for the corresponding version because this must be released now.